pkg/ux/output.go

```diff
@@ -32,7 +32,7 @@ func NewUserLog(log logging.Logger, userwriter io.Writer) {
 // PrintToUser prints msg directly on the screen, but also to log file
 func (ul *UserLog) PrintToUser(msg string, args ...interface{}) {
 	formattedMsg := fmt.Sprintf(msg, args...)
-	fmt.Fprintln(ul.writer, formattedMsg)
+	_, _ = fmt.Fprintln(ul.writer, formattedMsg)
```

Comment:
Huh, this looks a bit ugly, do we really need to do this?

Reply:
changed linting rules so this isn't needed